poe-api/src/poe.py

```diff
@@ -518,6 +550,27 @@ def send_message(self, chatbot, message, with_chat_break=False, timeout=20):
       message_id = message["messageId"]
 
       yield message
+    
+    # wait 2 senconds after sending the request
+    time.sleep(2)
+    self.request_count += 1
+
+    # send recv_post after receiving the last message
+    if self.request_count % 3 == 0:
```

Comment:
Not sure why this is a 1/3 chance even though the regular Poe web client sends a receive_POST after every message.